File: src/main/java/at/hannibal2/skyhanni/utils/jsonobjects/JacobContestsJson.java
package at.hannibal2.skyhanni.utils.jsonobjects;

import at.hannibal2.skyhanni.features.garden.CropType;
import com.google.gson.annotations.Expose;

import java.util.HashMap;
import java.util.List;
import java.util.Map;

public class JacobContestsJson {
    @Expose
    public Map<Long, List<CropType>> contestTimes = new HashMap<>();
}

File: src/main/java/at/hannibal2/hanni/data/jsonobjects/local/JacobContestsJson.kt
package at.hannibal2.hanni.data.jsonobjects.local

import at.hannibal2.hanni.data.jsonobjects.elitedev.EliteFarmingContest
import com.google.gson.annotations.Expose

data class JacobContestsJson(
    @Expose var knownContests: List<EliteFarmingContest> = listOf()
)
